server/connectors/README.md

```
@@ -0,0 +1,157 @@
+   - Products
+   - Disputes
+   - Payouts
+   - Customers
+5. Click **Create key** → reveal the secret value
+6. Add it to `.env.local` at the repository root:
+   ```bash
+   STRIPE_SECRET_KEY=rk_test_...   # or rk_live_... for production
+   ```
+
+> **Test mode is strongly recommended** for development. Stripe's test mode is a parallel-universe view of your account with fake money and test customers — completely isolated from live data. You can build and validate the connector without risk, then swap to a `rk_live_` key once ready for production.
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)